terraform/environment/region/ecs_task_reset_database.tf

```diff
@@ -18,7 +18,7 @@ locals {
     {
       name    = "reset-database",
       image   = local.images.api,
-      command = ["sh", "tests/Behat/reset-db.sh"],
+      command = ["sh", "scripts/reset_db_fixtures.sh"],
```

Comment:
Does the `local` argument need to be added for this to actually reset the db now?